README.md

```diff
@@ -4,63 +4,65 @@
 
 Our goal is to make it easy for XNA developers to create cross-platform games with extremely high code reuse.
 
-## Supported Platforms
+### Links
 
-* iOS (including Retina displays)
-* Android
-* Windows (OpenGL & DirectX)
-* Mac OS X
-* Linux
-* [Windows Store Apps](https://dev.windows.com/) (for Windows 8 and Windows RT)
-* [Windows Phone 8](http://dev.windowsphone.com)
-* [Windows Phone 8.1](http://dev.windows.com)
-* [Windows 10 UWP](http://dev.windows.com)
-* [OUYA](https://www.ouya.tv/), an Android-based gaming console
+ * Forum can be found at: [community.monogame.net](http://community.monogame.net/)
+ * Chat with the community at:&nbsp; [![Join the chat at https://gitter.im/mono/MonoGame](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/mono/MonoGame?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+ * Documentation is available at: [www.monogame.net/documentation/](http://www.monogame.net/documentation/)
+ * [Download Binaries](http://www.monogame.net/downloads/).
+ * Follow [@MonoGameTeam on Twitter](https://twitter.com/monogameteam).
 
-## Quick Start
+### Supported Platforms
```

Comment:
I think we have to still specify Windows 10 and Windows 8 separately.  Really anything that protobuild can generate should likely be in this list.